GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -109,26 +123,43 @@ public final class StoryboardParser {
       }
     }
   }
+
+  private func collectModules(initial: InitialScene?, scenes: Set<Scene>, segues: Set<Segue>) -> Set<String> {
+    var result = Set<String>()
+
+    if let module = initial?.customModule {
+      result.insert(module)
+    }
+    result.formUnion(Set<String>(scenes.flatMap { $0.customModule }))
+    result.formUnion(Set<String>(segues.flatMap { $0.customModule }))
+
+    return result
+  }
 }
 
 extension StoryboardParser.Scene: Equatable { }
 func == (lhs: StoryboardParser.Scene, rhs: StoryboardParser.Scene) -> Bool {
-  return lhs.storyboardID == rhs.storyboardID && lhs.tag == rhs.tag && lhs.customClass == rhs.customClass
+  return lhs.storyboardID == rhs.storyboardID &&
+    lhs.tag == rhs.tag &&
+    lhs.customClass == rhs.customClass &&
+    lhs.customModule == rhs.customModule
 }
 
 extension StoryboardParser.Scene: Hashable {
   var hashValue: Int {
-    return "\(storyboardID);\(tag);\(customClass)".hashValue
+    return "\(storyboardID);\(tag);\(customModule);\(customClass)".hashValue
   }
 }
 
 extension StoryboardParser.Segue: Equatable { }
 func == (lhs: StoryboardParser.Segue, rhs: StoryboardParser.Segue) -> Bool {
-  return lhs.segueID == rhs.segueID && lhs.customClass == rhs.customClass
+  return lhs.segueID == rhs.segueID &&
+    lhs.customClass == rhs.customClass &&
+    lhs.customModule == rhs.customModule
 }
 
 extension StoryboardParser.Segue: Hashable {
   var hashValue: Int {
-    return "\(segueID);\(customClass)".hashValue
```

Comment:
In case we decide to do that change, apply here as well (note that I'm not sure if `^` is a better hash or not, I've read it is, but also depends on how `String.hash` is computed by Apple, so… And I think it's cheaper than string interpolation anyway